test/functional/lua/luaeval_spec.lua

```diff
@@ -51,12 +51,12 @@ describe('luaeval()', function()
   end)
   describe('recursive lua values', function()
     it('are successfully transformed', function()
-      funcs.luaeval('rawset(_G, "d", {})')
-      funcs.luaeval('rawset(d, "d", d)')
+      command('lua rawset(_G, "d", {})')
```

Comment:
@ZyX-I The `funcs.luaeval` variant gives this error:

```
[ RUN      ] luaeval() recursive lua values are successfully transformed: ERR
test/functional/helpers.lua:99: E5101: Cannot convert given lua type
```

Reply:
Apparently it used to give the same error before, just ignored that.